### ISSUE TITLE
Add the ability to be notified for commits

### ISSUE THREAD

**Issue by itay** [tomasbjerre/pull-request-notifier-for-stash#23]
So this would change what the plugin would do, but it would be *great* to be able to be notified about commits as well (essentially pushes). There is already a plugin that does this:

https://bitbucket.org/aeffle/stash-http-get-post-receive-hook/

However, it doesn't do PRs. We currently use both, but it would be awesome if we could centralize on one plugin.

**Comment by tomasbjerre** [comment]
It would mean another set of variables. And the name of the plugin might have to be changed. I'll think about it, but I'm not sure its worth the effort. And I'm afraid the GUI may get harder to understand.

**Comment by mtakaki** [comment]
Does this issue involves notifying whenever a change is pushed to a pull request?

At the moment I'm having issues with it. I set it up to notify on `RESCOPED_FROM` and `RESCOPED_TO`, but I never get a notification when I push a change to the remote branch. Although it works fine when I add a comment to it (when I enable the `COMMENTED` option).

**Comment by itay** [comment]
@mtakaki we have it configured to notify on RESCOPED_FROM and it works just fine for us whenever a commit is pushed.

**Comment by mtakaki** [comment]
Interesting... It doesn't work at all for me. I even checked the jenkins git logs and I never get a notification of a change there. The most interesting thing is that other events work, except for `RESCOPED_*`.

**Comment by itay** [comment]
It's possible there is a bug in the later releases, we're a bit behind at the moment.

**Comment by mtakaki** [comment]
Which version are you using? Maybe I could try downgrading mine. Or are you behind on bitbucket version too?

**Comment by itay** [comment]
We are behind - currently on 4.1.3. I actually have a dev environment of the latest plugin and latest Bitbucket, let me try it there though.

**Comment by mtakaki** [comment]
@tomasbjerre Does it mean there's no bug with notifying commits pushed to pull requests?

**Comment by mtakaki** [comment]
Thanks, @itay! Really appreciate it. I'll be waiting for the result.

**Comment by tomasbjerre** [comment]
You may have found a bug @mtakaki.

Did you see the ***issues*** section in the [readme](https://github.com/tomasbjerre/pull-request-notifier-for-bitbucket/blob/master/README.md)? Perhaps you should open an issue.

**Comment by mtakaki** [comment]
@tomasbjerre I don't have access to the server logs, but I'll create an issue once I gather all the info. First I wanted to make sure this issue (#23) was not related to what I'm facing. And it would be great getting the result from @itay, so I can confirm if it's a real issue or just a configuration problem.

**Comment by tomasbjerre** [comment]
Its good to use the ${EVERYTHING_URL} when testing. As its the variable that is most likely to cause problems.

If you are using it @mtakaki , you may try not using it and instead use only the values you actually need.

**Comment by mtakaki** [comment]
I'm currently using `${EVERYTHING_URL}`, but it's working for other events... I'll give it a try anyway.

**Comment by mtakaki** [comment]
@tomasbjerre you're right. I removed `${EVERYTHING_URL}` and it worked. I guess I'll manually add the variables I need to the URL. Thank you both @tomasbjerre and @itay!

**Comment by tomasbjerre** [comment]
Om @mtakaki  I am very interested in your server logs. Would be nice if you can open an issue once you have them. Most likely you will find a stacktrace there!

**Comment by itay** [comment]
Yep, it seemed to be working for me as well on my dev instance (I think, having some other issue there).

@tomasbjerre it would be nice to maybe get a Gitter chat room for your repo. Not sure if you're interested in that or not.

**Comment by mtakaki** [comment]
@tomasbjerre I opened #138 for this issue.
